src/mudlet.cpp

```diff
@@ -3402,6 +3402,7 @@ QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1,
         // when saving/resyncing packages/modules - ends in a '/'
         return QStringLiteral("%1/.config/mudlet/moduleBackups/").arg(QDir::homePath());
     }
+    return QString();
```

Comment:
🤓 Given that the `switch()` is driven by an `enum` and all the `case` values __are__ included in the switch, please can we place a `Q_UNREACHABLE();` before the `return QString();` which will enforce this with an assert in debug builds...